contracts/oracle/VoltSystemOracle.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {Math} from "@openzeppelin/contracts/utils/math/Math.sol";
+import {Timed} from "./../utils/Timed.sol";
+import {Constants} from "./../Constants.sol";
+import {Initializable} from "@openzeppelin/contracts/proxy/utils/Initializable.sol";
+import {IVoltSystemOracle} from "./IVoltSystemOracle.sol";
+
+/// @notice contract that receives a fixed interest rate upon construction,
+/// and then linearly interpolates that rate over a 1 year period into the VOLT price
+/// after the oracle start time.
+/// Interest can compound annually.
+/// @author Elliot Friedman
+contract VoltSystemOracle is Timed, Initializable, IVoltSystemOracle {
```

Comment:
I think it'd be worth mentioning how and why Timing is used and maybe Initializable. 

something like, 
```
The contract inherits `startTime` from Timing which is set in the constructor and used to
calculate the current time delta.  `TIMEFRAME` is set as the Timing contract's `duration`.
And, the `aftertime` modifier gates the `compoundInterest` function.
```

Reply:
Removed both